docs/source/devguide/dev_assisted_package_creation.rst

```diff
@@ -0,0 +1,104 @@
+ .. dropdown:: Distribution Statement
+
+    | # # # Distribution Statement A. Approved for public release. Distribution unlimited.
+    | # # #
+    | # # # Author:
+    | # # # Naval Research Laboratory, Marine Meteorology Division
+    | # # #
+    | # # # This program is free software: you can redistribute it and/or modify it under
+    | # # # the terms of the NRLMMD License included with this program. This program is
+    | # # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+    | # # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+    | # # # for more details. If you did not receive the license, for more information see:
+    | # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+
+Developer Assisted Package Creation
+===================================
+
+Why We Assist With Package Creation
+-----------------------------------
+
+The GeoIPS Development team is committed to integrating a variety of geospatial
+processing routines into GeoIPS, or as a standalone packages that can interact with
+GeoIPS. Specifically we developers at CIRA are tasked with integrating proprietary
+software developed by researchers to work alongside GeoIPS. If you have access, see the
+`GeoIPS GLM Package <https://bear.cira.colostate.edu/overcast/geoips_glm>`_ for an
+example of a standalone package that implements GREMLIN and GLM
+(Global Lightning Mapper) routines working alongside GeoIPS.
+
+The purpose of integrating software like this is to elevate GeoIPS as a package that can
```

Comment:
This is great! Maybe we can re-word this, and move this to the beginning. Here's a good video about user/reader centric docs: https://www.youtube.com/watch?v=RpEb8XLH-1A

Reply:
I added some stuff to the beginning but kept that portion (although reworded) in the same section. I feel that purpose should be under the why, however I agreed on the point that the intro needed some work.